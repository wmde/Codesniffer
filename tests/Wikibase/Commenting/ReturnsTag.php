<?php

function noDocComment() {
}

/**
 * @return void This is fine.
 */
function validReturnTag() {
}

/**
 * @returns void This should be reported for using "@returns" with an "s" instead of "@return".
 */
function lowerCaseReturnsTag() {
}

/**
 * @Returns void Reporting should be case-insensitive.
 */
function upperCaseReturnsTag() {
}

/*
 * @returns void This is not a PHPDoc comment and should be ignored.
 */
function notADocComment() {
}
