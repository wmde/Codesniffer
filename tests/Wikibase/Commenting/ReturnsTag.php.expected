 13 | ERROR | Found @returns instead of @return
 19 | ERROR | Found @returns instead of @return
